modules/storages/spec/support/fixtures/vcr_cassettes/one_drive/files_info_query_one_not_found.yml

```diff
@@ -0,0 +1,93 @@
+---
+http_interactions:
+- request:
+    method: get
+    uri: https://graph.microsoft.com/v1.0/drives/b!dmVLG22QlE2PSW0AqVB7UOhZ8n7tjkVGkgqLNnuw2OBb-brzKzZAR4DYT1k9KPXs/items/01AZJL5PJTICED3C5YSVAY6NWTBNA2XERU?$select=id,name,fileSystemInfo,file,folder,size,createdBy,lastModifiedBy,parentReference
+    body:
+      encoding: US-ASCII
+      string: ''
+    headers:
+      Authorization:
+      - Bearer eyJ0eXAiOiJKV1QiLCJub25jZSI6IlJTdHpqTVJNSGcxOTFMcHJoOVcwRG94a2VkZGZOX2RIMm42dEw1Wm1TSFUiLCJhbGciOiJSUzI1NiIsIng1dCI6Ii1LSTNROW5OUjdiUm9meG1lWm9YcWJIWkdldyIsImtpZCI6Ii1LSTNROW5OUjdiUm9meG1lWm9YcWJIWkdldyJ9.eyJhdWQiOiIwMDAwMDAwMy0wMDAwLTAwMDAtYzAwMC0wMDAwMDAwMDAwMDAiLCJpc3MiOiJodHRwczovL3N0cy53aW5kb3dzLm5ldC80ZDQ0YmYzNi05YjU2LTQ1YzAtODgwNy1iYmYzODZkZDA0N2YvIiwiaWF0IjoxNjk1ODE5NTA1LCJuYmYiOjE2OTU4MTk1MDUsImV4cCI6MTY5NTgyNDEyNCwiYWNjdCI6MSwiYWNyIjoiMSIsImFpbyI6IkFXUUFtLzhVQUFBQUw3MnhpOVlPcDdOZjRJTEo1Z1EvUWJrcXI0ZVBPWlRnSU0xR1N2OVptcUZ1OUN0NlpnTWpMWERLQXVLQytrOU5nb0l1Y3pLUUhvOC9IVnh6RENlak5YTUlzQUxrdENoaEFOaTFwVnRCY3pCK3AzZ0ZrbVJqcjZ0SjVvbGJ1cktwIiwiYWx0c2VjaWQiOiIxOmxpdmUuY29tOjAwMDM0MDAxNjA2Q0VEMTciLCJhbXIiOlsicHdkIl0sImFwcF9kaXNwbGF5bmFtZSI6Ik9wZW5Qcm9qZWN0IERldiBBcHAiLCJhcHBpZCI6IjQyNjJkZjJiLTc3YmItNDljMi1hNWRmLTI4MzU1ZGE2NzZkMiIsImFwcGlkYWNyIjoiMSIsImVtYWlsIjoiZXNjaHViZXJ0Lm9wQG91dGxvb2suY29tIiwiZmFtaWx5X25hbWUiOiJTY2h1YmVydCIsImdpdmVuX25hbWUiOiJFcmljIiwiaWRwIjoibGl2ZS5jb20iLCJpZHR5cCI6InVzZXIiLCJpcGFkZHIiOiIyMDAzOmY3OmE3MDQ6MjAwOjkyOGU6ZjFhNToxNzdjOjliNWEiLCJuYW1lIjoiRXJpYyBTY2h1YmVydCIsIm9pZCI6IjBhMGQzOGE5LWE1OWItNDI0NS05M2ZhLTBkMmNmNzI3ZjE3YSIsInBsYXRmIjoiOCIsInB1aWQiOiIxMDAzMjAwMkVEMkNBMDBFIiwicmgiOiIwLkFRa0FOcjlFVFZhYndFV0lCN3Z6aHQwRWZ3TUFBQUFBQUFBQXdBQUFBQUFBQUFBSkFLOC4iLCJzY3AiOiJEaXJlY3RvcnkuUmVhZFdyaXRlLkFsbCBGaWxlcy5SZWFkV3JpdGUuQWxsIFNpdGVzLlJlYWQuQWxsIFNpdGVzLlJlYWRXcml0ZS5BbGwgVXNlci5SZWFkIHByb2ZpbGUgb3BlbmlkIGVtYWlsIiwic2lnbmluX3N0YXRlIjpbImttc2kiXSwic3ViIjoiWUFYYjFTaUR3RjNCUHFyN2lXWVhGT2YzMGxKSG9PeGtrYzQ4Z0R3c1Q1TSIsInRlbmFudF9yZWdpb25fc2NvcGUiOiJFVSIsInRpZCI6IjRkNDRiZjM2LTliNTYtNDVjMC04ODA3LWJiZjM4NmRkMDQ3ZiIsInVuaXF1ZV9uYW1lIjoibGl2ZS5jb20jZXNjaHViZXJ0Lm9wQG91dGxvb2suY29tIiwidXRpIjoiM2w0WlVBVXlwRWlqZHo3Z21WUWlBQSIsInZlciI6IjEuMCIsIndpZHMiOlsiODhkOGUzZTMtOGY1NS00YTFlLTk1M2EtOWI5ODk4Yjg4NzZiIiwiMTNiZDFjNzItNmY0YS00ZGNmLTk4NWYtMThkM2I4MGYyMDhhIl0sInhtc19zdCI6eyJzdWIiOiIwblA2VEVfQUhBeFM0dm5ibWFRcDFLZUEyUngzbXNBTXphRFQ4RTJ4RmpJIn0sInhtc190Y2R0IjoxNDA1NTE0ODYxLCJ4bXNfdGRiciI6IkVVIn0.J0z20sVW7ClLpsTnGZRJ_vULwXuXAtESGeQ9BidcFrR_k8Pb3Z8DIlgAMAhzK8glbfHVIk3hAOott7GCkEFZ3cWYw2HDvgeHb7urZMuxvMQ9bIrXS8YJLlUP5ATBdy0Xp42i5VzK4xvSxTaihJJh8Sf97MlQL6JRSnh2CEezBGopBKsUW3W6HsvPsypDltRRDBCW5tqD6cPWjv-GkWp0boWSCcP8UGx7ShTILRjIpN6DTB0MqaMgVh3clwYOEYviIGp7yjD4tl_9sJSqImDTXNipg0taTEcR6b8kyQuNWSgJJ0n_cygR3Lh7LbqNQ1ibnOjvA_FcjeYSiYXRBAlEFQ
```

Comment:
Should we add a rule to filter the bearer token? I'm not sure if these are short lived tokens- but perhaps safer to filter out by default?

https://benoittgt.github.io/vcr/#/configuration/filter_sensitive_data

Reply:
done, good idea!